### PR TITLE
Add document describing static and dynamic blocks.

### PR DESCRIPTION
**Pull request by mtias**
This is an important distinction that should be more visible.

### CODE REVIEW

**Comment by youknowriad** [comment]
This is kind of duplicate with this page https://wordpress.org/gutenberg/handbook/blocks/creating-dynamic-blocks/ 

Should we remove the old one maybe?

File: docs/block-static-dynamic.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+		// Rendering in PHP
+		return null;
+	},
+} );
+```
+{% ESNext %}
+```js
+// myblock.js
+
+const { el } = wp.element;
```
Comment (Soean) [inline docs/block-static-dynamic.md]: We should remove this line, as in #4213

**Comment by mtias** [comment]
@youknowriad the whole tutorial section needs to be cleaned up a bit later.

**Comment by aduth** [comment]
Closing as stale without response.

**Pull request closed without merging**